samples/tab-deeplink/nodejs/public/SubTask.js

```diff
@@ -0,0 +1,30 @@
+const path = require('path');
+const dotenv = require('dotenv');
+const ENV_FILE = path.join(__dirname, '.env');
+dotenv.config({ path: ENV_FILE });
+
+var task1Values = {"subEntityId": "topic1"}
+var jsontext1=JSON.stringify(task1Values);
+var task1Context=encodeURI(jsontext1);
```

Comment:
Create two methods to get deeplink string.

getDeepLinkToPersonalTab()
getDeepLinkToChannelTab()